src/interface/src/app/plan/create-scenarios/create-scenarios.component.ts

```diff
@@ -178,9 +169,15 @@ export class CreateScenariosComponent implements OnInit, OnDestroy {
     });
   }
 
-  ngOnDestroy(): void {
-    this.destroy$.next();
-    this.destroy$.complete();
+  pollForChanges() {
+    interval(POLLING_INTERVAL)
+      .pipe(untilDestroyed(this))
+      .subscribe(() => {
+        // only poll when scenario is pending
+        if (this.scenarioState === 'pending') {
```

Comment:
I believe we'll also want to poll if the scenario is 'running' as well ('pending' and 'running' both indicate the scenario doesn't currently have a result, but will eventually)